packages/client/components/MyDashboardTimeline.tsx

```diff
@@ -47,10 +46,8 @@ const MyDashboardTimeline = (props: Props) => {
       <TimelineFeed>
         <TimelineFeedItems>
           <ErrorBoundary>
-            <Suspense fallback={<TimelineLoadingEvents />}>
```

Comment:
@mattkrick I fixed the bouncing issue by removing this block. Not quite sure why that's the cause though.

Reply:
this tells me when a timeline event type is requested but the component hasn't been fetched yet, it will fallback to showing the mock for the entire list, not just that item!

to fix, instead of removing it here, add another suspense with fallback here: https://github.com/ParabolInc/parabol/blob/5a92a4605b066eb0bffb68f130b9ac10f8c3acf3/packages/client/components/TimelineEvent.tsx